indigo/distribution.yaml

```diff
@@ -1953,7 +1953,12 @@ repositories:
       tags:
         release: release/indigo/{package}/{version}
       url: https://github.com/yujinrobot-release/flask-ext-catkin-release.git
-      version: 0.1.0-0
+      version: 0.1.0-1
+    source:
+      type: git
+      url: https://github.com/asmodehn/flask-ext-catkin.git
+      version: 0.1.0
```

Comment:
Does it have to be a full git branch ?
I mean this is a tag : https://github.com/asmodehn/flask-ext-catkin/releases/tag/0.1.0
Should I specify something like 
```
source:
  version: master
```
?